servers/mlflowserver/mlflowserver/requirements.txt

```diff
@@ -1,7 +1,8 @@
-pyyaml==5.4.0
-requests<2.25.0
-mlflow<1.12.0
-pandas<1.2.0
+# Addresses CVE SNYK-PYTHON-PYYAML-590151
+PyYAML >= 5.4, < 5.5
+requests < 2.28.0
+mlflow < 1.24.0
+pandas < 1.2.0
 
```

Comment:
A part from PyYAML, how are these versions chosen ? MLFlow 1.24.0 is not even released

Why not unpinning the requirements altogether ? Is there any **real** dependency on a specific version of any of these packages ?